rbd/encryption.go

```diff
@@ -0,0 +1,117 @@
+// +build !octopus,!nautilus
+
+package rbd
+
+// #cgo LDFLAGS: -lrbd
+// /* force XSI-complaint strerror_r() */
+// #define _POSIX_C_SOURCE 200112L
+// #undef _GNU_SOURCE
+// #include <errno.h>
+// #include <stdlib.h>
+// #include <rados/librados.h>
+// #include <rbd/librbd.h>
+import "C"
+
+import (
+	"unsafe"
+)
+
+// formt is the encryption format type
+type format C.rbd_encryption_format_t
+
+// Possible values for format:
+// EncryptionFormatLUKS1: LUKS v1
+// EncryptionFormatLUKS2: LUKS v2
+const (
+	EncryptionFormatLUKS1 = format(C.RBD_ENCRYPTION_FORMAT_LUKS1)
+	EncryptionFormatLUKS2 = format(C.RBD_ENCRYPTION_FORMAT_LUKS2)
+)
+
+// EncryptionAlgorithm is the encryption algorithm
+type EncryptionAlgorithm C.rbd_encryption_algorithm_t
+
+// Possible values for EncryptionAlgorithm:
+// EncryptionAlgorithmAES128: AES 128bits
```

Comment:
Are these comments allowed? I expected the linters would complain about it. If not, I'm totally fine with it.